src/FbgemmFP16.cc

```diff
@@ -35,23 +35,15 @@ inline void PackA(int nrow, int ncol, const float* from, int ldim, float* to) {
 struct KernelInfo {
   using knl_ptr = funcptr_fp16;
   // optimized kernels to cover all cases
-  static constexpr array<knl_ptr, 15> kernel = {
+  static constexpr array<knl_ptr, 7> kernel = {
     {
       nullptr,
-      gemmkernel_1x1_AVX2_fA0fB0fC0,
-      gemmkernel_2x1_AVX2_fA0fB0fC0,
-      gemmkernel_3x1_AVX2_fA0fB0fC0,
-      gemmkernel_4x1_AVX2_fA0fB0fC0,
-      gemmkernel_5x1_AVX2_fA0fB0fC0,
-      gemmkernel_6x1_AVX2_fA0fB0fC0,
-      gemmkernel_7x1_AVX2_fA0fB0fC0,
-      gemmkernel_8x1_AVX2_fA0fB0fC0,
-      gemmkernel_9x1_AVX2_fA0fB0fC0,
-      gemmkernel_10x1_AVX2_fA0fB0fC0,
-      gemmkernel_11x1_AVX2_fA0fB0fC0,
-      gemmkernel_12x1_AVX2_fA0fB0fC0,
-      gemmkernel_13x1_AVX2_fA0fB0fC0,
-      gemmkernel_14x1_AVX2_fA0fB0fC0
+      gemmkernel_1x2_AVX2_fA0fB0fC0,
```

Comment:
Add comment 2 in ?x2 should be same as kernel_ncol_blocks?

Reply:
Added the comments.